llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -23975,7 +23919,10 @@ bool RISCVTargetLowering::lowerInterleavedStore(StoreInst *SI,
   // potentially under larger LMULs) because we checked that the fixed vector
   // type fits in isLegalInterleavedAccessType
   Value *VL = ConstantInt::get(XLenTy, VTy->getNumElements());
-  Ops.append({SI->getPointerOperand(), VL});
+  // All-ones mask.
+  Value *StoreMask = ConstantVector::getSplat(
+      VTy->getElementCount(), ConstantInt::getTrue(SVI->getContext()));
```

Comment:
For consistency should we also use Builder.getAllOnesMask here?

Reply:
oh right, I changed one placed but forgot about other occurrences. It's fixed now.